tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-card.html

```diff
@@ -188,10 +188,7 @@
       properties: {
         tag: String,
 
-        /**
-         * @type {Array<Object>}
-         */
-        dataToLoad: Array,
+        tagAndRuns: Object,
```

Comment:
Please document shape, especially since it’s _not_ what I would expect
from the name (which would be `{tag: string, runs: string[]}`).